recipes/imagemagick/all/conanfile.py

```diff
@@ -0,0 +1,399 @@
+            self._autotools.include_paths.remove("/usr/include/freetype2")
+        for include in self._autotools.include_paths:
+            if include.startswith("/usr/include/libpng"):
+                self._autotools.include_paths.remove(include)
+
+        args = ['--disable-openmp',
+                '--disable-docs',
+                '--with-perl=no',
+                '--with-x=no'
+                ]
+
+        if self.options.shared:
+            args.extend(['--enable-shared=yes', '--enable-static=no'])
+        else:
+            args.extend(['--enable-shared=no', '--enable-static=yes'])
+        args.append('--enable-hdri=yes' if self.options.hdri else '--enable-hdri=no')
+        args.append('--with-quantum-depth=%s' % self.options.quantum_depth)
+        args.append('--with-zlib=yes' if self.options.with_zlib else '--with-zlib=no')
+        args.append('--with-bzlib=yes' if self.options.with_bzlib else '--with-bzlib=no')
+        args.append('--with-lzma=yes' if self.options.with_lzma else '--with-lzma=no')
+        args.append('--with-lcms=yes' if self.options.with_lcms else '--with-lcms=no')
+        args.append('--with-openexr=yes' if self.options.with_openexr else '--with-openexr=no')
+        args.append('--with-jpeg=yes' if self.options.with_jpeg else '--with-jpeg=no')
+        args.append('--with-openjp2=yes' if self.options.with_openjp2 else '--with-openjp2=no')
+        args.append('--with-pango=yes' if self.options.with_pango else '--with-pango=no')
+        args.append('--with-png=yes' if self.options.with_png else '--with-png=no')
+        args.append('--with-tiff=yes' if self.options.with_tiff else '--with-tiff=no')
+        args.append('--with-webp=yes' if self.options.with_webp else '--with-webp=no')
+        args.append('--with-xml=yes' if self.options.with_xml2 else '--with-xml=no')
+        args.append('--with-freetype=yes' if self.options.with_freetype else '--with-freetype=no')
+        args.append('--with-utilities=yes' if self.options.utilities else '--with-utilities=no')
+        args.append('--with-djvu=yes' if self.options.with_djvu else '--with-djvu=no')
+        self._autotools.configure(args=args)
+
+        return self._autotools
+
+    def package(self):
+        with tools.chdir(self._source_subfolder):
+            env_build = self._build_configure()
+            env_build.install()
+
+        with tools.chdir(self.package_folder):
+            # remove undesired files
+            tools.rmdir(os.path.join("lib", "pkgconfig"))  # pc files
+            tools.rmdir("etc")
+            tools.rmdir("share")
+            tools.remove_files_by_mask("lib", "*.la")
+
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        if self._is_msvc:
+            self.copy(pattern="*CORE_*.lib", dst="lib", src=os.path.join('VisualMagick', 'lib'), keep_path=False)
+            self.copy(pattern="*CORE_*.pdb", dst="lib", src=os.path.join('VisualMagick', 'lib'), keep_path=False)
```

Comment:
We don't allow copying pdb's to the package atm.
The hooks should complain about this. (They aren't because Windows isn't built)

Also, pdb's should be copied to the `bin` folder (=same folder as the dlls/executables).